data/json/move_modes.json

```diff
@@ -0,0 +1,57 @@
+[
+  {
+    "id": "MM_NULL",
+    "type": "move_mode",
+    "name": "Null",
+    "flavor_text": "You should never see this."
+  },
+  {
+    "id": "MM_WALK",
+    "type": "move_mode",
+    "name": "Walk",
+    "can_haul": true,
+    "usable_while_mounted": true,
+    "speed": 1.0,
+    "display_colour": "light_gray",
+    "display_character": "W",
+    "minimum_required_legs": 0,
+    "flavor_text": "You start walking.",
```

Comment:
Translatable strings in JSON require separate code in "lang/extract_json_strings.py" to extract them for translation.